.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+   // Use IntelliSense to learn about possible attributes.
+   // Hover to view descriptions of existing attributes.
+   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
(suggestion; non-blocking): Comments in JSON files are not _technically_ allowed, but because this is for VS Code settings, we should be okay.